docs/recipes/integration-test/example-lh-auth.test.js

```diff
@@ -0,0 +1,183 @@
+/**
+ * @license Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+/**
+ * @fileoverview Example Jest tests for demonstrating how to run Lighthouse on an authenticated
+ * page as integration tests. See docs/recipes/auth/README.md for more.
+ */
+
+/** @typedef {import('lighthouse/types/lhr')} LH */
+
+const puppeteer = require('puppeteer');
+const lighthouse = require('lighthouse');
+const server = require('../auth/server/server.js');
+
+const CHROME_DEBUG_PORT = 8042;
+const SERVER_PORT = 8000;
```

Comment:
can you move all the 8000 port stuff to a diff port? i use it a lot for local stuff that i keep running, so i'd prefer to not conflict

10632 is random enough